examples/3d/msaa.rs

```diff
@@ -40,12 +39,14 @@ fn setup(
     // camera
     commands.spawn(Camera3dBundle {
         transform: Transform::from_xyz(-3.0, 3.0, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
+        msaa: Msaa::default(),
```

Comment:
This is the default value, but I think this should show the full struct like the original example

```suggestion
        msaa: Msaa { sample: 4 },
```